packages/material-ui/src/Select/SelectInput.js

```diff
@@ -305,6 +281,8 @@ const SelectInput = React.forwardRef(function SelectInput(props, ref) {
         anchorEl={displayRef.current}
         open={open}
         onClose={handleClose}
+        disableRestoreFocus
+        disableEnforceFocus={open}
```

Comment:
This looks like a bug. When we manually called displayRef.current.focus it got immediately blurred towards the Paper from the Popover. It shouldn't lock focus if it's closed anyway.